packages/api/controllers/hackathon_register_controller.go

```diff
@@ -152,3 +161,49 @@ func FormatTeamName(teamName string) string {
 func CompareTeamNames(passedTeamName string, storedTeamName string) bool {
 	return FormatTeamName(passedTeamName) == FormatTeamName(storedTeamName)
 }
+
+func SendEmailToNewParticipant(fullName string, email string) {
+
+	requestURL := os.Getenv("MAILING_URL")
+
+	type Mailer struct {
+		Receiver string `json:"receiver"`
+		Html     string `json:"html"`
+		Subject  string `json:"subject"`
+	}
+
+	var reqBody Mailer
+
+	reqBody.Html = "<h1>HELLO</h1>"
+	reqBody.Receiver = email
+	reqBody.Subject = "TEST"
+
+	fmt.Println(reqBody)
+	json_data, _ := json.Marshal(reqBody)
+
+	client := &http.Client{}
+
+	req, err := http.NewRequest(http.MethodPost, requestURL, bytes.NewBuffer(json_data))
+
+	if err != nil {
+		fmt.Println("could not create request to mailing service: ")
```

Comment:
Why do you keep going if the request creation failed? exit function

Reply:
Added return